README.md

```diff
@@ -1,10 +1,67 @@
-# Flexible Modeling System (FMS)
+# Modeling Framework: Flexible Modeling System (FMS)
+
+Today’s climate models simulate highly complex systems. In response to
+increasingly complex models, the climate community has developed tools and
+methodologies to facilitate the modeling process, developing modeling frameworks
+that provide infrastructure and tools to structure climate model codes and
+facilitate many common tasks (e.g., calendar management, grid generation, I/O).
+Such frameworks come with a number of advantages, including decreased model
+development time and increased compatibility of interfaces.
+
+The Flexible Modeling System (FMS) is a software environment that supports the
+efficient development, construction, execution, and scientific interpretation of
+atmospheric, oceanic, and climate system models. This framework allows
+algorithms to be expressed on a variety of high-end computing architectures
+using common and easy-to-use expressions of the underlying platforms, spanning
+distributed and shared memory, as well as high-performance architectures.
+Scientific groups at GFDL can develop new physics and new algorithms
+concurrently, and coordinate periodically through this framework.
+
+![FMS Framework](docs/images/FMS.gif)
+
+Modeling frameworks for the construction of coupled models, made from
+independent model components, are now prevalent across this field. FMS was one
+of the first frameworks to be developed — since the advent of the Cray T3E in
+1998 — and is still in use and under development today, using new architectures
+and new algorithms.
+
+## What is FMS
 
 The Flexible Modeling System (FMS) is a software framework for supporting the
-efficient development, construction, execution, and scientific interpretation
-of atmospheric, oceanic, and climate system models.
+efficient development, construction, execution, and scientific interpretation of
+atmospheric, oceanic, and climate system models.  FMS comprises of the
+following:
+
+1. A software infrastructure for constructing and running atmospheric, oceanic,
+and climate system models. This infrastructure includes software to handle
+parallelization, input and output, data exchange between various model grids,
+orchestration of the time stepping, makefiles, and simple sample run scripts.
+This infrastructure should largely insulate FMS users from machine-specific
+details.
+1. A standardization of the interfaces between various component models.
```

Comment:
A standardization of the interfaces between various component models, including ~~S~~ software...